lib/conditionals.md

```diff
@@ -5,8 +5,8 @@ instructor_notes: Feel free to re-organize the headings (or add/remove headings)
 
 # What is a conditional?
 
-Replace me with your response...
+A conditional establishes what should happen when the code incounters a specific input. For example, a conditional could say if name == shannon, then that person is great.
```

Comment:
EDIT: Conditionals can ask “is this equal to this?” but they can also ask “is this more than this? less than this? is it capitalized?” and a slew of other questions. (After reading all this information, I realize that I interact with conditionals most obviously when I’m trying to make “safe” passwords for every website on earth). Conditionals are how you make loops, but gives the code an out to make sure that loop is not infinite. It how to make your program seem to have a conversation with you, by making it possible to respond to input appropriately and not just with a set order of answers.